src/pages/Login/Login.js

```diff
@@ -1,10 +1,112 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate, Link } from 'react-router-dom';
+import './Login.scss';
 
 const Login = () => {
+  const [userInfo, setUserInfo] = useState({
+    email: '',
+    password: '',
+  });
+
+  const navigate = useNavigate();
+
+  const idCondition = userInfo.email.includes(('@', 4) && '.com');
+  const pwCondition = userInfo.password.length >= 5;
+
+  const handleIdInput = e => {
+    setUserInfo({
+      ...userInfo,
+      email: e.target.value,
+    });
+  };
+
+  const handlePasswordInput = e => {
+    setUserInfo({
+      ...userInfo,
+      password: e.target.value,
+    });
+  };
```

Comment:
```suggestion
  const updateUserInfo = e => {
    const {name, value} = e.target;
    setUserInfo({...userInfo, [name] : value});
  };
```
이렇게 하나의 함수로 사용할 수도 있습니다 :)